### PR TITLE
fix govet errors

### PR DESCRIPTION
**Pull request by mfojtik**
For some reason govet (./hack/verify-govet.sh) locally yelling on me for these (and exiting 1), but I don't see them failing in our CI. Anyway, this PR fixes them.

UPDATE: It seems like the OSX version of `go tool vet` and Linux version of `go tool vet` produces different results

/cc @deads2k

### CODE REVIEW

File: pkg/oc/cli/admin/migrate/storage/storage.go [hunk 1]
```diff
@@ -362,7 +362,7 @@ func (t *tokenLimiter) getDuration(n int) time.Duration {
 	reservation := t.rateLimiter.ReserveN(now, n)
 	if !reservation.OK() {
 		// this should never happen but we do not want to hang a worker forever
-		glog.Errorf("unable to get rate limited reservation, burst=%d n=%d limiter=%#v", t.burst, n, *t.rateLimiter)
+		glog.Errorf("unable to get rate limited reservation, burst=%d n=%d", t.burst, n)
```
Comment (mfojtik) [inline pkg/oc/cli/admin/migrate/storage/storage.go]: `call of glog.Errorf copies lock value: github.com/openshift/origin/vendor/golang.org/x/time/rate.Limiter contains sync.Mutex`

File: pkg/image/trigger/buildconfigs/buildconfigs.go [hunk 2]
```diff
@@ -144,7 +144,7 @@ func (r *buildConfigReactor) ImageChanged(obj runtime.Object, tagRetriever trigg
 			continue
 		}
 		if p.Paused {
-			glog.V(5).Infof("Skipping paused build on bc: %s/%s for trigger: %s", bc.Namespace, bc.Name, t)
+			glog.V(5).Infof("Skipping paused build on bc: %s/%s for trigger: %+v", bc.Namespace, bc.Name, t)
```
Comment (sttts) [inline pkg/image/trigger/buildconfigs/buildconfigs.go]: the trigger is full of pointers. Does it actually look helpful?

Comment (mfojtik) [reply pkg/image/trigger/buildconfigs/buildconfigs.go]: it is up to @bparees but I guess it will at least print the type ;-)

**Comment by tnozicka** [comment]
I don't mind the fixups; the thing is unless those are enforced by CI you are going to be doing its job and sending PRs to fix it.

**Comment by tnozicka** [comment]
I think our issue is that go vet doesn't run at all in the CI, nor locally! (/cc @stevekuznetsov)
a) it takes about 6s which is too suspicious. Upstream:
```
SUCCESS  verify-govet.sh	287s
```
b) if you add `-v` here you will see all the compile failures but calling the equivalent `go tool vet ...` exits 0 even so \O/ https://github.com/tnozicka/origin/blob/1a8b960ce025a09f5e3e244c0836c9850c9ce3ca/hack/verify-govet.sh#L27

running pure `go vet ./pkg/...` should make you aware in how bad state this is...

File: pkg/image/trigger/buildconfigs/buildconfigs.go (hunk 2 above)
Comment (bparees) [reply pkg/image/trigger/buildconfigs/buildconfigs.go]: probably fine.

**Comment by stevekuznetsov** [comment]
That script may have been written when `go vet` was available only in the `go tool vet` form -- it's pretty old. A PR to update to use `go vet` would be useful

**Pull request merged**